server/services/caseloadService.ts

```diff
@@ -32,5 +49,12 @@ export default class CaseloadService {
       .filter(offender => !offender.indeterminateSentence)
       .filter(offender => !offender.paroleEligibilityDate)
       .filter(offender => offender.legalStatus !== 'DEAD')
+      .filter(offender => offender.status && offender.status.startsWith('ACTIVE'))
+      .filter(offender => !offender.releaseDate || moment().isBefore(moment(offender.releaseDate, 'yyyy-MM-DD')))
+      .filter(offender => !offender.homeDetentionCurfewEndDate)
+      .filter(offender => {
+        const existingLicence = existingLicences.find(licence => licence.nomisId === offender.nomsNumber)
+        return !existingLicence
+      })
```

Comment:
Is this filter right? Any chance of two licences for the same nomisId, in different statuses? Probably right.